### ISSUE TITLE
Configuration problem

### ISSUE THREAD

**Issue by Happie661** [u-blox/c209-aoa-tag#2]
I have problems with AT commands. I have the tag software installed and C211 flashed. But I cant configure with s-center since the console window do not allow me to paste code, which I can done it before. I would like to use s-center other than other software for safety. What can be done in this situation? Thank you.

**Comment by perssonmagnus** [comment]
Hello, it should be possible to paste AT commands in s-center for sending to the C209 tag. 

Two things as noted in the XPLR-AOA kits user guide,https://www.u-blox.com/en/docs/UBX-21004616, ch. 4.2.1:

1) The AT interface is only available for 10 seconds after reset
2) You need to open the COM port with settings 115200 bps, and no flow control

**Comment by Happie661** [comment]
Thank you. How about C211 board? How to configure after software installed? Thank you.

**Comment by perssonmagnus** [comment]
Hi,

This is also described in the XPLR-AOA kits user guide, chapter 3.3.4 for the UART settings (for this board flow control should be enabled), and the available AT commands are described in Appendix A.

Best Regards,
Magnus

**Comment by Happie661** [comment]
I configured the COM port as instructed but cannot input the AT command in the console window. Anything I need to pressed on the board to do the configuration? Thank you.

**Comment by perssonmagnus** [comment]
Hello,

Assuming that you have flashed the C209 with the SW there should not be anything else, besides the fact that the AT interface is only available for 10 seconds after a reset. Is the SW running with the blue LED blinking to indicate that the board is advertising?

**Comment by Happie661** [comment]
Yes I have flashed the C209. When the tag is powered up, the C211 automatically detect the location of the tag and the blue LED blinks. The reason I want to enter the AT command is because I want to see the angle calculation as seen in chapter 5 (+UUDF). When I opened the s-center with the C211 and tag powered up I saw hex numbers in the console window, but I want to see the angle. Is there any way I can obtain a more meaningful information rather than a group of hex number that I dont know the meaning? Thank you.

**Comment by perssonmagnus** [comment]
Ah, OK. So you are connecting to the C211. 

OK, if the tag is detected by the C211 anchor you should get +UUDF events. s-center can be configured to display incoming data in UTF-8 or HEX (by pressing the "Advanced" button), did you accidentally change that?

![image](https://user-images.githubusercontent.com/42796456/143406221-f97bd0f4-e192-4089-b98b-43a53bbc9550.png)

Do you get the same result if using another terminal emulator, like Teraterm?

**Comment by Happie661** [comment]
I was advised not to use other terminal emulator from my senior so I didn't try out other software.
![before tag](https://user-images.githubusercontent.com/65290404/143409194-0fc8c70d-af67-4d2e-af6f-fd393f958398.JPG)
![after tag](https://user-images.githubusercontent.com/65290404/143409219-55cb7a23-d7ee-4172-ba58-3f631877f7eb.JPG)
This is the picture before and after the tag is powered up. The hex numbers are output onto the console window non-stop while the tag is advertising.

**Comment by Happie661** [comment]
I don't have other components attached with the C211. Is it necessary to connect an Arduino board or EKV-ODIN-W2?

**Comment by jakkra** [comment]
You did not change the "Incoming Encoding" to utf-8 in the screenshot. Also I would recommend using https://www.putty.org/ to view the +UUDF events, but it works with s-center also.

> Is it necessary to connect an Arduino board or EKV-ODIN-W2?
No not needed.

**Comment by Happie661** [comment]
![utf8_pic](https://user-images.githubusercontent.com/65290404/143411525-be7801ef-0bb0-4bee-9e05-9a6ec2ad71cd.JPG)
That's what I see when I choose UTF-8. Totally unreadable. Does the left down corner writing "Frame Error" matters?

**Comment by perssonmagnus** [comment]
The "Frame error" usually indicates that you are running on the wrong baud rate, but I see that you are starting off with the correct 115200 rate. 

Please try to do  factory reset of the C211 board by holding down SW1 while resetting the board (hold SW1 and press+release RESET. Keep holding SW1 for 10 seconds) as described in chapter 3.3.2.2 of https://www.u-blox.com/en/docs/UBX-21004616.

**Comment by Happie661** [comment]
I actually have two C211 boards. Just now when I tried the second board, it works normally (i.e. as shown in Chapter 5, showing the +UUDF). However, when I tried to factory reset the first board, the board still appears unreadable code in UTF8. Would there be a problem in the first C211 board?

**Comment by perssonmagnus** [comment]
It could very well be a problem with the board, but it would be a bit surprising since I assume you have flashed the u-connectLocate SW on the board and not noticed any problem then? You could also try to reflash the SW using the procedure described in section 3.3.3 of the user guide linked above. 

If this doesn't help please go to https://portal.u-blox.com/s/ and create a support case for further help.

**Comment by Happie661** [comment]
I thought it might be my problem of not configuring it correctly since 3.3.4 said AT commands are needed to configured the C211 but I can't enter any. I have tried to factory reset the board and reflashed it a few days ago as mentioned in the last comment, but now when I tried to factory reset again(cause last time didn't work), it does not seem to be able to be reset.

Thank you I will create a support case on this.
